go/ngql/query.go

```diff
@@ -56,21 +62,27 @@ func NewRootQueryObject(rootValue types.Value, tm *TypeMap) *graphql.Object {
 // NewContext creates a new context.Context with the extra data added to it
 // that is required by ngql.
 func NewContext(vr types.ValueReader, tm *TypeMap) context.Context {
-	return context.WithValue(context.WithValue(context.Background(), vrKey, vr), tmKey, tm)
+	return context.WithValue(context.Background(), vrKey, vr)
```

Comment:
It seems like we should be able to get rid of Context entirely and put everything on TypeConverter? Where "everything" is the ValueReader at this point.